platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/utils/DataUnit.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging.utils;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+public enum DataUnit {
```

Comment:
Nice, I wanted to have this talk.
I saw those classes before adding this one, but they are not part of the config API or visible to the logging module. It would be great to have in swirlds-base instead of commons.

Moving the existing ones to the base seemed out-of-scope for this task, but the newer ones might be too simple to become general-purposes classes...
Options were to handle it this way and to create a ticket to build better support in the future. Or do it all in the same ticket. 
What is your preference?

Reply:
We decided to create a follow-up task and perform this changes next